packages/sdk/src/oracle/utils.ts

```diff
@@ -1,39 +1,91 @@
-import { State } from "./types/state";
+import { State, RequestState, Flag } from "./types/state";
```

Comment:
these utilities have been changed to be a key:value type of flag system. Flags are now strings to make it more intuitive when logging.